src/api/views.py

```diff
@@ -19,6 +19,7 @@
 
 
 DOMAIN_FILE_URL = "https://raw.githubusercontent.com/cisagov/dotgov-data/main/current-full.csv"
```

Comment:
Can you remove this and any uses of this while your in the code

Reply:
@abroddrick sure! I'm not as familiar with where this is used but is current-full no longer referenced in manage.get.gov?

Reply:
Nope it isn't